HOW_TO_RUN.md

```diff
@@ -0,0 +1,983 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
In my opinion, we don't need this file.

We need only basic instructions on how to run the app locally. This can be put in `Readme.md` itself.
Most of the information in this file is unnecessary for this app.

Also, it's not a good idea to maintain this. it's better to put a link to the documentation to the documentation for create react app. https://create-react-app.dev/

Reply:
Ok, I will remove it